perception_dataset/rosbag2/autoware_msgs.py

```diff
@@ -129,18 +140,29 @@ def get_category_name(classification: List[ObjectClassification]) -> str:
 
     scene_annotation_list: List[Dict[str, Any]] = []
     for obj in msg.objects:
-        obj: Union[DetectedObject, TrackedObject]
+        obj: Union[
+            AutowareAutoDetectedObject,
+            AutowareAutoTrackedObject,
+            AutowareDetectedObject,
+            AutowareTrackedObject,
+        ]
         pose = obj.kinematics.pose_with_covariance.pose
 
-        if isinstance(obj, DetectedObject):
+        print(
+            AutowareAutoDetectedObject,
+            AutowareAutoTrackedObject,
+            AutowareDetectedObject,
+            AutowareTrackedObject,
+        )
+        if isinstance(obj, AutowareAutoDetectedObject) or isinstance(obj, AutowareDetectedObject):
```

Comment:
we can make this shorten.
```suggestion
        if isinstance(obj, (AutowareAutoDetectedObject, AutowareDetectedObject)):
```

Reply:
Thanks. Fixed: https://github.com/tier4/tier4_perception_dataset/pull/110/commits/41ada9161c023e4cb2e22865820e7377708cb80b